### PR TITLE
Add BGB gameboy emulator

### PR DESCRIPTION
**Pull request by amir16yp**
Add the BGB gameboy/gameboy color emulator. it was confirmed to be working with sound on 32 bit and 16bit color modes, with sound set to WaveOut, on stable and on nightly by members of the reactos discord.
image

### CODE REVIEW

File: bgb.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = BGB - GB/GBC emulator
+Version = 1.5.9
+License = Freeware
+Description = A GameBoy/GameBoy color emulator, lets you play gb/gbc games on your PC.
+Category = 4
+URLSite = https://bgb.bircd.org
+URLDownload = https://bgb.bircd.org/bgb.zip
+SHA1 = 58c3d937d9f0d3e113e5112d959b7f14f3b1bc2a
+SizeBytes = 468214
```
Comment (learn-more) [inline bgb.txt:10]: Please ensure that the last line is an empty line.

**Review by amir16yp — COMMENTED** [review]
did it fix it now?

**Review by amir16yp — COMMENTED** [review]
i tested in rapps already

**Comment by amir16yp** [comment]
well, while this pull request was waiting to be merged bgb got an update. i updated the package and synced the master into my fork

**Review by learn-more — APPROVED** [review]

**Pull request merged**